Integrations/VMRay/integration-VMRay.py

```diff
@@ -0,0 +1,636 @@
+""" GLOBAL PARAMS """
+API_KEY = demisto.params()["api_key"]
+SERVER = (
+    demisto.params()["server"][:-1]
+    if (demisto.params()["server"] and demisto.params()["server"].endswith("/"))
+    else demisto.params()["server"]
+)
+
+SERVER += "/rest/"
+USE_SSL = not demisto.params().get("insecure", False)
+HEADERS = {"Authorization": "api_key " + API_KEY}
+
+# Remove proxy
+if not demisto.params().get("proxy"):
+    del os.environ["HTTP_PROXY"]
+    del os.environ["HTTPS_PROXY"]
+    del os.environ["http_proxy"]
+    del os.environ["https_proxy"]
+
+""" HELPER DICTS """
+SEVERITY_DICT = {
+    "malicious": "Malicious",
+    "suspicious": "Suspicious",
+    "not_suspicious": "Good",
+    "blacklisted": "Blacklisted",
+    "whitelisted": "Whitelisted",
+    "unknown": "Unknown",
+}
+
+DBOTSCORE = {
+    "Malicious": 3,
+    "Suspicious": 2,
+    "Good": 1,
+    "Blacklisted": 3,
+    "Whitelisted": 1,
+    "Unknown": 0,
+}
+
+""" HELPER FUNCTIONS """
+
+
+def http_request(method, url_suffix, body=None, params=None, files=None):
+    """ General HTTP request.
+    Args:
+        method: (str) "GET", "POST", "DELETE' "PUT"
+        url_suffix: (str)
+        body: (dict)
+        params: (dict)
+        files: (tuple, dict)
+
+    Returns:
+        dict: response json
+    """
+
+    url = SERVER + url_suffix
+    r = requests.request(
+        method,
+        url,
+        json=body,
+        params=params,
+        headers=HEADERS,
+        files=files,
+        verify=USE_SSL,
+    )
+    if r.status_code not in {200, 201, 202, 204}:
```

Comment:
are all these status codes needed ? can they all be returned ? are they all legit?